aspnetcore/tutorials/first-mongo-app.md

```diff
@@ -334,7 +335,7 @@ To satisfy the preceding requirements, make the following changes:
 
 ## Additional resources
 
-* [View or download sample code](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/tutorials/first-mongo-app/samples) ([how to download](xref:index#how-to-download-a-sample))
+* [View or download sample code](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/tutorials/first-mongo-app/samples/8.x/BookStoreApi) ([how to download](xref:index#how-to-download-a-sample))
```

Comment:
There is no 9.x version of the docs. If desired I can create them.

Reply:
Thanks.  If you want to that's great.  This topic is on my list of many that need an update.  You could update it afterwards in a new PR or this one.  Up to you.